dev-libs/sway/sway-1.0_rc2-r1.ebuild

```diff
@@ -65,36 +70,37 @@ src_prepare() {
 
 src_configure() {
 	local emesonargs=(
-		$(meson_use bash-completion bash-completions)
-		$(meson_use fish-completion fish-completions)
+		-Dman-pages=$(usex man enabled disabled)
+		-Dtray=$(usex tray enabled disabled)
+		-Dxwayland=$(usex X enabled disabled)
 		$(meson_use wallpapers default-wallpaper)
+		$(meson_use fish-completion fish-completions)
 		$(meson_use zsh-completion zsh-completions)
-		"-Dtray=$(usex tray enabled disabled)"
-		"-Dxwayland=$(usex X enabled disabled)"
+		"-Dbash-completions=true"
 		"-Dwerror=false"
 	)
+
 	if use swaybar || use swaybg; then
 		emesonargs+=("-Dgdk-pixbuf=enabled")
 	else
 		emesonargs+=("-Dgdk-pixbuf=disabled")
 	fi
-	if [[ ${PV} != 9999 ]]; then
-		emesonargs+=("-Dsway-version=${PV}")
-	fi
```

Comment:
still missing from rc2

Reply:
rc2 and 9999 must not have these 3 lines. `sway-version` has been removed on >=1.0_rc2.
These lines remains only in rc1 since this version still uses `sway-version`.
This causes rc1 ebuild to be different from rc2 and 9999, but i think it's fine: that's the *difference* between rc1 and later versions.

Reply:
ah, ok, ya, that's good then.